### DISCUSSION TITLE
Massive decrease of climb rate because of speed oscilation in higher Flight Levels

### DISCUSSION THREAD

**Discussion by FGrabi** [flybywiresim/aircraft#5925] [Help]
I'm using always the actual DEV Version of FBW with Pilot2ATC for the "communication" and REX WeatherForce 2020. During the last 3-4 weeks with flights like EFHK-EDDB -> Flight Plan from SIMBRIEF -> FL360 -> Stepclimbs -> REX real weather I'm facing the following problem:
P2A advises me to climb in steps (FL140-FL220-FL280-FL360) what I do in managed mode. Everything is fine and "normal" with a climb rate up to 2100 fpm. As soon as the plane climbs in higher altitudes (mostly around FL260-FL280), the climb rate decreases dramaticly to 500fpm or lower. This happens, because the managed speed slows down and the AP levels the plane (0 fpm) to "restore" the speed back to the computed CLMB-Speed. If the speed is "restored" the plane start slowly climbing again (500-700fpm) and often the speed starts decreasing again. P2ATC calls every time to climb and maintain the adviced FL and/or asked to expedite the climb. It takes "ages" to climb to the CRZ FL.

Did someone knows this problem and can give me an advice or hint?

I'm using TA1600 equipment. The throttle is calibrated in EFB and works perfect during start and flight. The joystick has a deadzone. Nothing changed in the sensitivity settings.

Thanks for your help and sorry for my bad english :-(

**Comment by ghost** [comment]
This should be solved with a new improved speed/mach law which is currently in the testing phase.
